### PR TITLE
Update User Stories and Development Workflow guides

### PR DESCRIPTION
**Pull request by Shiun**
Update How We Write User Stories to include example.

Update Development Workflow Guide to be less rails-centric.

### CODE REVIEW

File: process/development_workflow.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ Work on a story usually begins by using GitFlow in Tower (or on the git command
 
  
 
-In most cases, all the code for a given story is contained in a single commit. (As an exception, when including an unrelated refactoring or other code change in a story, it usually makes sense to add an additional commit.) The same commit includes tests that use rspec to cover every line of controller and model code. Additional rspecs may be needed to cover any calls to underlying service layers or APIs. If the feature or bug contains a testable UI change, we also write a cucumber test to exercise it.
+In most cases, all the code for a given story is contained in a single commit. (As an exception, when including an unrelated refactoring or other code change in a story, it usually makes sense to add an additional commit.) The same commit includes new tests that cover the added code. If the feature or bug contains a testable UI change, we also write a cucumber test to exercise it.  Unit tests ensure the base logic works as intended.  Functional tests ensures the application works as intended.
```
Comment (JeffreyATW) [inline process/development_workflow.md:37]: `s/Functional tests ensures/Functional tests ensure`

File: process/development_workflow.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ Work on a story usually begins by using GitFlow in Tower (or on the git command
...
 
-We use usually use brakeman to enforce code coverage. If brakeman code coverage or quality issues arise when running foreman run rake ci:build, it’s helpful to open coverage/index.html to see what lines of code are triggering the brakeman error.
+Use code quality scanners, such as [brakeman](https://brakemanscanner.org/) to ensure new code does not introduce vulnerabilities.  Use code coverage tool, such as [simplecov](https://github.com/simplecov-ruby/simplecov) to ensure new code is appropriately tested.
```
Comment (JeffreyATW) [inline process/development_workflow.md:39]: `s/Use code coverage/Use a code coverage`

File: process/development_workflow.md [hunk 3]
```diff
@@ -102,7 +102,7 @@ Good commit log:
...
 
-Just because there were three or four sort-of-related things that could be done at once does not mean they should be lumped into one PR. The problem that arises in this situation is that one of the multiple changes is blocked for some reason, and since the changes are all now linked together, all changes are effectively blocked.
+Just because there were three or four sort-of-related things that could be done at once does not mean they should be lumped into one PR. The potential for one of the changes to block the other changes in the PR increases.  This is why we try out best to break stories into individual chunks of work.  And as such, changes should be as small individual chunks of change.
```
Comment (JeffreyATW) [inline process/development_workflow.md:105]: `s/And as such/As such`

File: process/how_we_write_user_stories.md [hunk 4]
```diff
@@ -35,12 +35,27 @@ Describe the outcome the user achieves.
...
 The description in your story should provide additional context and clarify what will be tested when the story is delivered.
 
-Consider trying the “Given-When-Then” formula, a template intended to guide the writing of acceptance tests. It is likely that the developer will write automated tests based on your story, some tests may match the criteria you use to accept the work. 
+Consider trying the “Given-When-Then” formula, a template intended to guide the writing of acceptance tests ([Gherkin](https://cucumber.io/docs/gherkin/)). It is likely that the developer will write automated tests based on your story, some tests may match the criteria you use to accept the work. 
 
 * Given [some context, this is the pre-condition to the activity]
 * When [some action is carried out]
 * Then [a particular change or set of changes you expect due to the specified action]
 
+Here's an example of a login in story.
```
Comment (JeffreyATW) [inline process/how_we_write_user_stories.md:44]: `s/login in/login`

File: process/how_we_write_user_stories.md [hunk 5]
```diff
@@ -35,12 +35,27 @@ Describe the outcome the user achieves.
...
+>_Feature: As a user I want to sign in and see my orders_  
+>
+>__Scenario: User supplies correct username and password__  
+>_Given that I am on the log in page_  
+>_When I enter my username and password correctly_  
+>_And click 'Sign in'_  
+>_Then I am taken to my dashboard showing my orders_  
+>
+>__Scenario: User does NOT supplies correct username and password__
```
Comment (JeffreyATW) [inline process/how_we_write_user_stories.md:53]: `s/NOT supplies/NOT supply`

**Review by JeffreyATW — CHANGES_REQUESTED** [review]
Looks great! Some small spelling corrections.

**Comment by Shiun** [comment]
A lot of grammar errors 🙈.  Pushing up changes now

File: process/development_workflow.md [hunk 6]
```diff
@@ -102,7 +102,7 @@ Good commit log:
...
 
-Just because there were three or four sort-of-related things that could be done at once does not mean they should be lumped into one PR. The problem that arises in this situation is that one of the multiple changes is blocked for some reason, and since the changes are all now linked together, all changes are effectively blocked.
+Just because there were three or four sort-of-related things that could be done at once does not mean they should be lumped into one PR. The potential for one of the changes to block the other changes in the PR increases.  This is why we try out best to break stories into individual chunks of work.  As such, changes should be as small individual chunks of change.
```
Comment (JeffreyATW) [inline process/development_workflow.md:105]: `s/try out best/try our best`

**Review by JeffreyATW — CHANGES_REQUESTED** [review]

**Review by JeffreyATW — APPROVED** [review]

**Pull request merged**